extensions/blocks/calendly/utils.js

```diff
@@ -1,5 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import { REGEX } from './index';
+
 export const getURLFromEmbedCode = embedCode => {
-	const url = embedCode.match( /(?<!\.)calendly\.com.+?([^"']*)/i );
+	const url = embedCode.match( REGEX );
 	if ( url ) {
 		return 'https://' + url[ 0 ];
```

Comment:
I think maybe the regex here just needs to be [pulled from master](https://github.com/Automattic/jetpack/blob/master/extensions/blocks/calendly/utils.js#L2)

Reply:
Ah yeah sorry that was my bad, i just did copy/paste.